### DISCUSSION TITLE
Support for Unmount?

### DISCUSSION THREAD

**Discussion by miko** [jfyne/live#57] [Ideas]
Hello,
is it possible to implement Unmount()? Say you want to display the number of currently active users on the page, or you want to free some server resources after the last connection (socket) is closed.

**Comment by jfyne** [comment]
I've seen this, on holiday, I will get back to you soon

**Reply by miko** [reply]
Looking at the `engine.go` code, I would add `disconnectHandler: func(sock Socket)` to BaseEngine and `HandleDisconnect(handler DisconnectHandler)` to the Engine interface, and then call it within DeleteSocket().

**Reply by miko** [reply]
Thanks for merging in https://github.com/jfyne/live/pull/63

**Reply by jfyne** [reply]
Tell me if you are able to do what you need with it
